node/src/Cardano/Node/API.hs

```diff
@@ -204,77 +230,6 @@ instance Core.HasSlottingVar SettingsCtx where
     slottingVar =
         lens settingsCtxSlottingVar (\s t -> s { settingsCtxSlottingVar = t })
 
-applyUpdate :: Handler NoContent
-applyUpdate =
-    throwError err500 { errBody = "This handler is not yet implemented." }
-
-{-
--- from Cardano.Wallet.API.Internal.Handlers
-applyUpdate :: PassiveWalletLayer IO -> Handler NoContent
-applyUpdate w = liftIO (WalletLayer.applyUpdate w) >> return NoContent
-
--- from Cardano.WalletLayer.Kernel.hs
--- ...
-        , applyUpdate          = Internal.applyUpdate         w
--- ...
-
--- from Cardano.Wallet.WalletLayer.Kernal.Internal
--- | Apply an update
---
--- NOTE (legacy): 'applyUpdate', "Pos.Wallet.Web.Methods.Misc".
---
--- The legacy implementation does two things:
---
--- 1. Remove the update from the wallet's list of updates
--- 2. Call 'applyLastUpdate' from 'MonadUpdates'
---
--- The latter is implemented in 'applyLastUpdateWebWallet', which literally just
--- a call to 'triggerShutdown'.
---
--- TODO: The other side of the story is 'launchNotifier', where the wallet
--- is /notified/ of updates.
-applyUpdate :: MonadIO m => Kernel.PassiveWallet -> m ()
-applyUpdate w = liftIO $ do
-    update' (w ^. Kernel.wallets) $ RemoveNextUpdate
-    Node.withNodeState (w ^. Kernel.walletNode) $ \_lock -> do
-      doFail <- liftIO $ testLogFInject (w ^. Kernel.walletFInjects) FInjApplyUpdateNoExit
-      unless doFail
-        Node.triggerShutdown
-
-So this is actually a bit of a problem. `update'` is updating the *wallet*
-acid-state database to remove information about the presence of an update. So we
-need to coordinate the information about wallet *and* node executable
-distribution separately?
```

Comment:
cf comment above + slack conversation. This is easier than it seems.

Reply:
OK, good to know. I'll revert deleting this part of the API.